app/service/form_task_list_service.rb

```diff
@@ -124,12 +124,26 @@ def make_form_live_section
     if Pundit.policy(@current_user, @form).can_make_form_live?
       section[:rows] = make_form_live_section_tasks
     else
-      section[:body_text] = I18n.t("forms.task_list_create.make_form_live_section.if_not_permitted.body_text")
+      section[:body_text] = make_form_live_section_body_text
     end
 
     section
   end
 
+  def make_form_live_section_body_text
+    return I18n.t("forms.task_list_create.make_form_live_section.if_not_permitted.body_text") unless FeatureService.enabled?(:groups)
+    return I18n.t("forms.task_list_create.make_form_live_section.group_not_active.group_admin.body_text", upgrade_path: group_upgrade_url) if !@form.group.active? && Pundit.policy(@current_user, @form).can_administer_group?
+    return I18n.t("forms.task_list_create.make_form_live_section.group_not_active.group_editor.body_text", group_members_path: group_members_path(@form.group)) unless @form.group.active?
+
+    I18n.t("forms.task_list_create.make_form_live_section.user_cannot_administer.body_text", group_members_path: group_members_path(@form.group)) unless Pundit.policy(@current_user, @form).can_administer_group?
+  end
+
```

Comment:
We could put the two different @form.group.active? cases into a method - do you reckon this makes it any easier to read? Feel free to ignore if not
```suggestion
  def make_form_live_section_body_text
    return I18n.t("forms.task_list_create.make_form_live_section.if_not_permitted.body_text") unless FeatureService.enabled?(:groups)
    return inactive_group_message unless @form.group.active?

    I18n.t("forms.task_list_create.make_form_live_section.user_cannot_administer.body_text", group_members_path: group_members_path(@form.group)) unless Pundit.policy(@current_user, @form).can_administer_group?
  end

  def inactive_group_message
    return I18n.t("forms.task_list_create.make_form_live_section.group_not_active.group_admin.body_text", upgrade_path: group_upgrade_url) if Pundit.policy(@current_user, @form).can_administer_group?

    I18n.t("forms.task_list_create.make_form_live_section.group_not_active.group_editor.body_text", group_members_path: group_members_path(@form.group))
  end
```

Reply:
Yeah I like this suggestion too, it was getting too complicated at a glance I think